### PR TITLE
MPV MPRIS plugin fix

### PR DESCRIPTION
**Pull request by harrisonthorne**
mpv and the mpv-mpris plugin weren't working together out-of-the-box for me, so these edits fix that.

I've confirmed on my end that these changes work :)

The changes made are based on the documentation for mpv-mpris [here](https://github.com/hoyon/mpv-mpris).

### CODE REVIEW

File: srcpkgs/mpv-mpris/template [hunk 1]
```diff
@@ -16,6 +16,6 @@ checksum=673aff031e7cc741edea68d7b4b0103d7b031d4a55755abb9e1be5dd4ec4e969
 
 do_install() {
 	vlicense LICENSE
-	vmkdir usr/lib/${pkgname}
-	vinstall mpris.so 0644 usr/lib/${pkgname}
+	vmkdir etc/mpv/scripts
+	vinstall mpris.so 0644 etc/mpv/scripts/
```
Comment (ericonr) [inline srcpkgs/mpv-mpris/template:20]: Does this enable it by default? In my setup I have `~/.config/mpv/scripts/mpris.so -> /usr/lib/mpv-mpris/mpris.so`, and it just works. Having it enabled for everyone wouldn't be ideal for a multiuser setup or similar.

Comment (flexibeast) [reply srcpkgs/mpv-mpris/template:20]: Also, would this result in errors for people who aren't using a D-Bus bus?

Comment (harrisonthorne) [reply srcpkgs/mpv-mpris/template]: What does that do exactly (I added vdoc anyways)? I was thinking something more along the lines of using `INSTALL.msg`

Comment (ericonr) [reply srcpkgs/mpv-mpris/template]: It adds the README to the package, so people can hunt for how to enable it in `/usr/share/doc/mpv-mpris`.

**Comment by ericonr** [comment]
Closed in c26a75635468cb20c6d6fafc7bb66095e9f58975 and its parent.

**Pull request closed without merging**